Integrate pharses from database into visualization

nurellina opened the pull request:
Here is the code with props. 
Followed this tutorial https://www.youtube.com/watch?v=PHaECbrKgs0&ab_channel=TheNetNinja
Why it is not working?

nurellina committed:
Integrate pharses from database into visualization

aeschi reviewed (changes requested):
- the index.js is fine

- in show-phrases.js I couldn't access the database, but I worked with an array I wrote (phrasesArray) instead. So if you can get the database to export the phrases as an array (which already worked for you I think) you can simply exchange the props content:
     - from: <Visualization phrases={phrasesArray} />
     - to: <Visualization phrases={phrases} />

- in Visualisation.js I added props as an argument to my const Visualization arrow function:
    - const Visualization = props => {....
    - it's the same as const Visualization = (props) => {....
    - or the way you did it, but without the { } so: function Visualization( props ) {
- now I can use the array as props.phrases in my Visualization.js 

    - you only use { } when you directly refer to the props name; in our case phrases
    - so your version would have worked with: function Visualization({ phrases }) {
    - and then we could just refer to it as phrases in Visualisation.js (not props.phrases)

I'll commit what I have and you can fix the database thingy.

Thanks :)

aeschi committed:
sending array from show-phrases to visualization with props

- only the array from the database is missing
- might be an access problem

aeschi committed:
fix visualisation firebase connection

- visualisation now appends the phrases from firebase if available
- fixed the visualisations piling up underneath each other

aeschi committed:
Merge branch 'master' into visualization-integration
